src/plugins/initiatives/initiative.js

```diff
@@ -1,9 +1,26 @@
 // @flow
 
-import {type NodeAddressT} from "../../core/graph";
+import {type NodeAddressT, NodeAddress} from "../../core/graph";
+import {initiativeNodeType} from "./declaration";
 
 export type URL = string;
 
+// Composite ID, used as input for NodeAddressT.
+export opaque type InitiativeId = string[];
```

Comment:
why not `$ReadOnlyArray<string>`? Though I admit it doesn't matter much when the type is opaque.

We might later want it to be an opaque subtype of $ROA<string>, but I don't know that we need to do that now.

Reply:
As you mentioned, it didn't occur to me because the opaque type doesn't make it necessary.
We have a grand total of 2 lines of code reading this array. :smile:
Happy to take a follow-up to change it though.